src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -281,6 +281,42 @@ Usage:
 
 `;
 
+
+const UPGRADE_OPTIONS = `
+Usage:
+
+'upgrade' is a noobaa-core command that will return the exported metrics of the deployed NooBaa system.
```

Comment:
Why do we need the "noobaa-core" in the help of the user?
```suggestion
'upgrade' is a noobaa-cli command that will return the exported metrics of the deployed NooBaa system.
```

What does it mean "the exported metrics"?

Reply:
it's a left over copied from metrics, removed